acceptance/bundle/artifacts/whl_dynamic/output.txt

```diff
@@ -0,0 +1,170 @@
+my_test_code/dist/my_test_code-0.0.1-py3-none-any.whl
+prebuilt/other_test_code-0.0.1-py3-none-any.whl
+
+=== Verify contents of the zip file
+>>> find.py --expect 1 .databricks/.*my_test_code.*whl
+src/__init__.py
+src/__main__.py
+my_test_code-0.0.1+[TIMESTAMP_NS].dist-info/METADATA
+my_test_code-0.0.1+[TIMESTAMP_NS].dist-info/WHEEL
+my_test_code-0.0.1+[TIMESTAMP_NS].dist-info/entry_points.txt
+my_test_code-0.0.1+[TIMESTAMP_NS].dist-info/top_level.txt
+my_test_code-0.0.1+[TIMESTAMP_NS].dist-info/RECORD
+
+=== Expecting 2 patched wheels in libraries section in /jobs/create
+>>> jq -s .[] | select(.path=="/api/2.1/jobs/create") | .body.tasks out.requests.txt
+[
+  {
+    "environment_key": "test_env",
+    "python_wheel_task": {
+      "entry_point": "run",
+      "package_name": "my_test_code"
+    },
+    "task_key": "ServerlessTestTask"
+  },
+  {
+    "existing_cluster_id": "0717-132531-5opeqon1",
+    "for_each_task": {
+      "inputs": "[1]",
+      "task": {
+        "existing_cluster_id": "0717-132531-5opeqon1",
+        "libraries": [
+          {
+            "whl": "/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/artifacts/.internal/my_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl"
+          }
+        ],
+        "python_wheel_task": {
+          "entry_point": "run",
+          "package_name": "my_test_code"
+        },
+        "task_key": "SubTask"
+      }
+    },
+    "libraries": [
+      {
+        "whl": "/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/artifacts/.internal/my_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl"
+      },
+      {
+        "whl": "/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/artifacts/.internal/other_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl"
+      }
+    ],
+    "python_wheel_task": {
+      "entry_point": "run",
+      "package_name": "my_test_code"
+    },
+    "task_key": "TestTask"
+  }
+]
+
+=== Expecting 2 patched wheels to be uploaded
+>>> jq .path
+"/api/2.0/workspace-files/import-file/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/artifacts/.internal/my_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl"
+"/api/2.0/workspace-files/import-file/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/artifacts/.internal/other_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl"
+"/api/2.0/workspace-files/import-file/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/files/my_test_code/dist/my_test_code-0.0.1-py3-none-any.whl"
+"/api/2.0/workspace-files/import-file/Workspace/Users/[USERNAME]/.bundle/python-wheel/default/files/prebuilt/other_test_code-0.0.1-py3-none-any.whl"
+
+=== Updating the local wheel and deploying again
+>>> [CLI] bundle deploy
+Building my_test_code...
+Uploading .databricks/bundle/default/patched_wheels/my_prebuilt_whl_other_test_code/other_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl...
+Uploading .databricks/bundle/default/patched_wheels/my_test_code_my_test_code/my_test_code-0.0.1+[TIMESTAMP_NS]-py3-none-any.whl...
```

Comment:
This is an internal path. Not great, but I'm not sure if special-casing this is better...